.travis.yml

```diff
@@ -7,7 +7,7 @@ matrix:
     - rvm: 2.3.4
     - rvm: 2.2
     - rvm: 2.1
-    - rvm: jruby-9.1.8.0
+    - rvm: jruby-9.1.10.0
       env:
```

Comment:
Thanks @olleolleolle, what about adding `jruby-1.9.1` instead? So we can keep track of the latest major version?